toolchain/check/testdata/class/generic/call.carbon

```diff
@@ -55,12 +55,28 @@ library "no_conversion";
 
 class Class(T:! type, N:! i32) {}
 
-// CHECK:STDERR: fail_no_conversion.carbon:[[@LINE+6]]:8: ERROR: Cannot implicitly convert from `i32` to `type`.
+// --- call_in_nested_return_type.carbon
+
+class Outer(T:! type) {
+  class Inner(U:! type) {
+    fn A() -> Outer(T) {
+      return {};
+    }
+    fn B() -> Outer(U) {
+      return {};
+    }
+    fn C() -> Inner(T) {
+      return {};
+    }
+    fn D() -> Inner(U) {
+      return {};
+    }
+  }
+}
+
+// CHECK:STDERR: call_in_nested_return_type.carbon:[[@LINE+3]]:8: ERROR: Name `Class` not found.
 // CHECK:STDERR: var a: Class(5, i32*);
-// CHECK:STDERR:        ^~~~~~
-// CHECK:STDERR: fail_no_conversion.carbon:[[@LINE-5]]:1: Initializing parameter 1 of function declared here.
-// CHECK:STDERR: class Class(T:! type, N:! i32) {}
-// CHECK:STDERR: ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// CHECK:STDERR:        ^~~~~
 var a: Class(5, i32*);
```

Comment:
Did this wind up in `call_in_nested_return_type.carbon` by accident? It seems like it still belongs in `fail_no_conversion.carbon`

Reply:
Yes, oops. Fixed, thanks!